src/routes/login/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import replace from 'lodash/replace';
+import upperCase from 'lodash/upperCase';
+
+export default function Button({ name, buttonLoginHandler }) {
```

Comment:
This confuses a bit. If it's gonna be the generic button component, call its handler `onClick` or `handleClick` and move it to `src/components`. Otherwise, if it's specific to your route, rename it to `LoginButton`